src/main/java/com/gmail/inverseconduit/datatype/Room.java

```diff
@@ -0,0 +1,144 @@
+package com.gmail.inverseconduit.datatype;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.gmail.inverseconduit.BotConfig;
+import com.gmail.inverseconduit.SESite;
+import com.gmail.inverseconduit.utils.SourceCodeParser;
+
+public class Room {

```

Comment:
Beeing unsure what you mean I will ask for safety: You mean that I can leave out the getters and setters because the attributes are not supposed to be changed once created? Or did I misunderstood something there?